VTEX - Default Values API.json

```diff
@@ -0,0 +1,431 @@
+        "in": "header",
+        "name": "X-VTEX-API-AppKey",
+        "description": "Unique identifier of the [API key](https://developers.vtex.com/docs/guides/api-authentication-using-api-keys)."
+      },
+      "appToken": {
+        "type": "apiKey",
+        "in": "header",
+        "name": "X-VTEX-API-AppToken",
+        "description": "Secret token of the [API key](https://developers.vtex.com/docs/guides/api-authentication-using-api-keys)."
+      },
+      "VtexIdclientAutCookie": {
+        "type": "apiKey",
+        "in": "header",
+        "name": "VtexIdclientAutCookie",
+        "description": "[User token](https://developers.vtex.com/docs/guides/api-authentication-using-user-tokens), valid for 24 hours."
+      }
+    },
+    "parameters": {
+      "Content-Type": {
+        "name": "Content-Type",
+        "in": "header",
+        "description": "Type of the content being sent.",
+        "required": true,
+        "style": "simple",
+        "schema": {
+          "type": "string",
+          "example": "application/json"
+        }
+      },
+      "Accept": {
+        "name": "Accept",
+        "in": "header",
+        "description": "HTTP Client Negotiation Accept Header. Indicates the types of responses the client can understand.",
+        "required": true,
+        "style": "simple",
+        "schema": {
+          "type": "string",
+          "example": "application/json"
+        }
+      }
+    },
+    "schemas": {
+      "DefaultValuesRequest": {
+        "type": "object",
+        "description": "Request body containing default values configuration for an organizational unit.",
+        "required": [
+          "id",
+          "defaultValues"
+        ],
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Unique identifier of the default values document."
+          },
+          "defaultValues": {
+            "type": "array",
+            "description": "An array of default value objects.",
+            "items": {
+              "type": "object",
+              "description": "Default value object defining entity-specific defaults.",
+              "required": [
+                "entity",
+                "entityValueId"
+              ],
+              "properties": {
+                "entity": {
+                  "type": "string",
+                  "description": "The name of the entity. Possible values:  <br>•`address/shipping`  <br>•`address/billing`  <br>•`creditCard`  <br>•`customFieldValues/{{customFieldId}}` for regular custom fields (e.g., PO Number, Cost Center, Release) where `{{customFieldId}}` is the `DocumentId` from Create Custom Field Settings v2  <br>•`customFieldValues/{{customFieldId}}/{{auxId}}` for Location custom fields where `{{customFieldId}}` is the `DocumentId` returned by the Create Custom Field Settings v2 endpoint (e.g., `89f1da93-6917-4cbf-894e-1f1399682826c`) and `{{auxId}}` is the address ID (only for Location custom fields)."
```

Comment:
We won't use `v2` in the public docs because the first version of custom fields was only internal and never made it to alpha. So the endpoint name is actually Create Custom Field Settings.